security/monitoring.adoc

```diff
@@ -0,0 +1,202 @@
+[[security-monitoring]]
+= Monitoring Cluster Events and Logs
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[security-monitoring-intro]]
+== Introduction
+
+In addition to security measures mentioned in other sections of this
+guide, the ability to monitor and audit an {product-title} cluster is an
+important part of safeguarding the cluster and its users against
+inappropriate usage.
+
+There are two main sources of cluster-level information that
+are useful for this purpose: events and logs.
+
+[[security-monitoring-events]]
+== Cluster Events
+
+Cluster administrators are encouraged to familiarize themselves with the _Event_ resource
+type and review a xref:../dev_guide/events.adoc#events-reference[list of events] to
+determine which events might be of interest. Depending on the master controller and plugin
+configuration, there are typically more potential event types than listed here.
+
+Events are associated with a namespace, either the namespace of the
+resource they are related to or, for cluster events, the *_default_*
+namespace. The *_default_* namespace is likely to hold the most relevant
```

Comment:
"likely" is  vague - we should be more confident in our instructions - does it hold relevant events or not?

Reply:
Depends on what you're interested in / what you consider relevant. This namespace holds events relevant to the entire cluster. Other namespaces may also have events that are of interest to the user, depending on what they're interested in. Perhaps reword not to assert which are "most" relevant.